### ISSUE TITLE
Some word lists don't appear to have https, and PGP signed files don't appear to be validated

### ISSUE THREAD

**Issue by ghost** [akheron/diceware.py#7]
It would be nice to see any of 

 - https on word lists (probably out of your hands), 
 - sha256sum check or pgp signature validation after download but before use

Thanks for your consideration.

**Comment by akheron** [comment]
You're right. To be honest, this script shouldn't download anything from anywhere itself.
